packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -652,20 +662,20 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 			// Summarize so that the above data store is unreferenced.
 			const { summarizer: summarizer1 } = await loadSummarizer(container);
 			const { summaryVersion } = await ensureSynchronizedAndSummarize(summarizer1);
+			summarizer1.close();
 
-			// Send a trailing op and close the summarizer.
 			testDataObject._root.set("key", "value");
-			summarizer1.close();
-			await delay(sweepTimeoutMs + 10);
+			const container2 = await provider.loadTestContainer(testContainerConfig);
+			await provider.ensureSynchronized();
+			// Close the first container to simulate scenario where it closes before its session expires.
+			container.close();
```

Comment:
Does this prevent the "unexpected delete" log or whatever, about the local thing getting deleted?

Reply:
Yes, it does.